### PR TITLE
Book animation & Painting Puzzle

### PR DESCRIPTION
**Pull request by Cavekitty16**

### CODE REVIEW

**Comment by zoogies** [comment]
Stuff to do:
- sound effect
- particle effects?
- should it make the player freeze in place while transitioning? (i think maybe yes)

File: game/Assets/Scripts/PaintingPuzzle/PaintingPuzzleObject.cs [hunk 1]
```diff
@@ -0,0 +1,12 @@
+using UnityEngine;
+using UnityEngine.SocialPlatforms.Impl;
+
+public class PaintingPuzzleObject : MonoBehaviour
```
Comment (Daniel-Frederick) [inline game/Assets/Scripts/PaintingPuzzle/PaintingPuzzleObject.cs:4]: This entire file can be deleted. It does nothing and PaintingPuzzleScore.cs already has all of the functionality for the painting puzzle.

File: game/Assets/Scripts/PaintingPuzzle/PaintingPuzzleScore.cs [hunk 2]
```diff
@@ -0,0 +1,50 @@
...
+    private GameObject painting;
+    private Rigidbody rb;
+    private BoxCollider boxCollider;
+    private static int puzzleScore = 0;
+    [SerializeField] private string[] Tags;  // Tags to check for collision
+
+    private void Start()
+    {
+        painting = gameObject;  // Store the reference to the GameObject this script is attached to
+        rb = GetComponent<Rigidbody>(); // Store reference to GameObjects RigidBody
```
Comment (Daniel-Frederick) [inline game/Assets/Scripts/PaintingPuzzle/PaintingPuzzleScore.cs:14]: There is nothing wrong with this file but for future reference please delete these useless comments. Comments should only explain **WHY** the code is there, never what it is doing.

File: game/Assets/Scripts/CharacterController.cs [hunk 3]
```diff
@@ -85,10 +85,11 @@ void Update()
...
+         * it is in TimeBook now
          */
-        if (gameInput.GetInteractClick())
-        {
-            timeTravel.SwitchTimePeriod();
-        }
+        //if (gameInput.GetInteractClick())
+        //{
+        //    timeTravel.SwitchTimePeriod();
+        //}
```
Comment (Daniel-Frederick) [inline game/Assets/Scripts/CharacterController.cs:93]: if you don't need this delete it

**Pull request merged**